src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -755,4 +755,66 @@ void testSAmodeACAllBranchMonitoredFlowTransfer() {
         assertEquals(1.66, brl14l13.getP1(), 1e-2);
         assertEquals(0.66, brl14l13.getFlowTransfer(), 1e-2);
     }
+
+    @Test
+    void testSAWithRemoteSharedControl() {
```

Comment:
This is not working, look at the logs.

Reply:
I fixed it. We now have to unit test post contingency reactive power using monitors.